### ISSUE TITLE
sendTitle or sendSubTitle doesnt work.

### ISSUE THREAD

**Issue by akaSnapy** [ServerSoftwareArchive/Tesseract#1]
SendTitle function doesnt work. Its.it gonna be fixed?
-->
* Tesseract:1.2.0
* PHP:7.0.15
* Server OS:CentOS
* Game version: 1.0.6
* Server host:Choopa LCC

**Comment by ZaydPE** [comment]
you need to be using the default minecraft resources to see this, I don't think it shows with texture packs

**Comment by akaSnapy** [comment]
I'm. Not using any kind of texutre packs

**Comment by NFGamerMC** [comment]
Please expand on this issue

**Comment by jdongel2004** [comment]
Template..?

**Comment by NFGamerMC** [comment]
Yes, when you create an issue it gives you the correct layout that we need.

**Comment by AppleDevelops** [comment]
Are you trying to send it upon join?

**Comment by LordJoshie** [comment]
I used that Regions plugin, and edited the plugin to instead of doing sendPopup it's sendTitle, and it does that when you enter/exit an area.... works flawless.

**Comment by erkam2002** [comment]
Its dont work upon join was me why?

**Comment by AppleDevelops** [comment]
To get it to work on join, schedule a delayed task.

**Comment by erkam2002** [comment]
ok

**Comment by erkam2002** [comment]
thx
